apps/web/src/components/use-command-extension-config.ts

```diff
@@ -3,7 +3,10 @@ import { CommandExtensionProps } from "../lib/tiptap/types";
 
 export function useCommandExtensionConfig<
   Props extends Record<string, string | number | null>
->(props: CommandExtensionProps<Props>) {
+>(
+  props: CommandExtensionProps<Props>,
+  transform?: <T extends keyof Props>(key: T, value: Props[T]) => Props[T]
```

Comment:
This TypeScript give me memories from Vietnam

Reply:
`transform` -> `propsTransform`?

Reply:
Fixed!